bin/gollum-migrate-tags

```diff
@@ -6,6 +6,24 @@ require 'rubygems'
 
 REPO = ARGV[0] || Dir.pwd
 
+def set_const(const, val)
+  already_defined = begin
+    Object.const_get(const)
+    true
+  rescue NameError
+    false
+  end
+  Object.const_set(const, val) unless already_defined
+end
+
+def const_true(const)
```

Comment:
Not sure I follow this logic at all I'm afraid. Need some context.

Reply:
I wanted to avoid having to do `bla if defined?(CONST) && CONST` all the time. But again, that may be better than this metacode.

Reply:
Found a nicer way to do all this :)